src/mount/sauna_client.cc

```diff
@@ -2359,7 +2359,17 @@ BytesWritten write(Context &ctx, Inode ino, const char *buf, size_t size, off_t
 		fileinfo->mode = IO_WRITE;
 		fileinfo->data = write_data_new(ino);
 	}
-	err = write_data(fileinfo->data,off,size,(const uint8_t*)buf);
+
+	Attributes attr;
+	attr.fill(0);
+	if (usedircache) {
+		gDirEntryCache.lookup(ctx, ino, attr); 
+	}
+	struct stat stbuf;
+	attr_to_stat(ino, attr, &stbuf);
+	size_t currentSize = stbuf.st_size;
+
+	err = write_data(fileinfo->data,off,size,(const uint8_t*)buf, currentSize);
```

Comment:
Consider adding a space between comma separated parameters to comply with code style.

Reply:
Done.